xds/src/test/java/io/grpc/xds/internal/sds/SdsProtocolNegotiatorsTest.java

```diff
@@ -351,54 +343,6 @@ public void nullTlsContext_nullFallbackProtocolNegotiator_expectException() {
     }
   }
 
-  @Test
-  public void noMatchingFilterChain_expectException() {
```

Comment:
If this test is removed, where is this scenario tested?

Reply:
I think it is covered in, e.g. `noFilterChainMatch_noDefaultSslContext` in `FilterChainProtocolNegotiatorTest`.
We changed the way serverSdsHandler consumes the config, so this test no longer valid.